### DISCUSSION TITLE
Clocks do not change after switching between power modes

### DISCUSSION THREAD

**Discussion by LuisManChelo** [seerge/g-helper#192] [Q&A] [answered]
Hi. I wanted to give this software a test and has been a blast so far, however I feel bugged due to the fact that switching modes won't edit the clocks from my GPU and CPU.
Here, the CPU is sitting at 1.4 GHz, however it should be reaching 3.0 Ghz without the boost. After some digging I realized my power plan from Windows was not changing with their respective power mode. Is there a way to make the power mode change with its respective Windows setting? Thank you.

![image](https://user-images.githubusercontent.com/96843297/229715620-baf7f5e1-c768-44ff-ac70-10c5750b3df6.png)
![image](https://user-images.githubusercontent.com/96843297/229715729-cac154ac-c709-44e2-8cdd-d1c326d4fe43.png)

**Comment by seerge** [comment, marked as answer]
Hello, app works with currently selected power plan (as in windows 11 for example they are de-facto deprecated and it's only Balanced available by default for users).

As mentioned in readme - What app DOES - it sets a "power mode". It has same effect as power plan (in terms of adjusting performance). In win 10 it looks like this i think:

![47bf4eb0-8083-4a5e-b157-801849a6cccd](https://user-images.githubusercontent.com/5920850/229729600-93f44c81-cd4b-4506-8202-6855485c22ff.png)

You should select balanced (as you have mentioned) and clocks will reach whatever they are supposed to reach there :)
